### ISSUE TITLE
Unfulfilled portion of order exceeds account balance?

### ISSUE THREAD

**Issue by sangaman** [lightninglabs/pool#171]
I've encountered a weird situation and it's unclear to me what happened.

I opened an account (unfortunately I couldn't find the history for the tx and amount where that happened) and made an offer for 594 units, which was about all my available balance in the account. That order was partially filled for 530 units, leaving me 64 units remaining. I can see the tx in the blockchain with the 530 unit output for the new channel, as well as the output for my "new" account.

The problem is, I'd expect this new account to have ~64 units available, and in fact my order stayed active for quite some time with 64 `units_unfulfilled`.  Eventually I canceled the order and tried to place a new one. But my account only has ~20 units available as per the `value` and `available_balance` output when I list accounts, trying to place a new order for 64 units fails due to insufficient balance.

What happened to the remaining ~44 units?

**Comment by Roasbeef** [comment]
Which version are you on? Did you set a min channel amount? Did you set the same fee rate and rate for both orders?

**Comment by sangaman** [comment]
Version `0.3.2-alpha` through one of the LiT releases. I did set min channel amounts for both orders, it was min 50 units for the first order and for the second order I'm trying to do min 64 (so all 64 units in one batch). I'm trying to set a slightly lower fee rate for this new order.

**Comment by Roasbeef** [comment]
You should update to the latest version. Some bug fixes to both account
management and available balance estimation were included.

**Comment by Roasbeef** [comment]
You can also look at `pool auction leases` and/or `pool orders list --verbose`. The verbose flag will show you all matching operations related to that order, it's possible you missed a cleared batch perhaps.

**Comment by sangaman** [comment]
I updated to `0.3.3-alpha`.

The output of `auction leases` shows just the one lease for 530 units. The output of `orders list --verbose` is blank after having restarted poold to update, but I ran it before restarting and it showed a handful of events like `OrderCreated` `OrderMatch` and `OrderUpdate` but they all showed a `units_filled` of 530, I didn't see anything indicating another order or match.

My account balance still shows the same, and it matches the tx output in the blockchain so it doesn't appear to be mistaken. Anything else I should try or any more info I could provide to help figure out what's going on?

**Comment by Roasbeef** [comment]
Which version were you running before you updated?

**Comment by Roasbeef** [comment]
You need to add the archived flag to see the existing set of orders since you cancelled it.

**Comment by sangaman** [comment]
OK I think this makes sense now and your hunch is correct. I see in the blockchain a prior transaction linked to my account for 54,94x,xxx sats (549 units and change) and then the order I placed was for 594 units (probably a typo), which could have gone through if the amount of the order wasn't checked against the available funds. That would then explain my current account balance. Closing this issue as I believe it's been resolved, thank you for the help.
